web/assets/js/Chat/Widget.js

```diff
@@ -159,6 +159,18 @@ class WidgetElement extends HTMLElement
 
     _onChatAssigned(event)
     {
+        this.dispatchEvent(
```

Comment:
Only works because the widget is listening for `ConnectFour.ChatAssigned`. This dependency should also be worked on. However, another pull request, not this one.